docs/src/test/java/docs/http/javadsl/server/testkit/MyAppService.java

```diff
@@ -40,13 +38,10 @@ public Route createRoute() {
 
   public static void main(String[] args) throws IOException {
     final ActorSystem system = ActorSystem.create();
-    final ActorMaterializer materializer = ActorMaterializer.create(system);
 
     final SimpleServerApp app = new SimpleServerApp();
 
-    final ConnectHttp host = ConnectHttp.toHost("127.0.0.1");
-
-    Http.get(system).bindAndHandle(app.createRoute().flow(system, materializer), host, materializer);
+    Http.get(system).newServerAt("127.0.0.1", 8080).bind(app.createRoute());
```

Comment:
Much nicer. Maybe add newlines?

Reply:
It's already shorter than before and there are so many similar instances of this, so I'd rather keep it like this than touching it again right now. The whole documentation will need an overhaul so I'll try to fix those cosmetics in a run next week.